.envrc

```diff
@@ -14,7 +14,8 @@ PYTHON=$(type -p python3.4 || type -p python3 || type -p python)
 
 layout python $PYTHON
 
-export DATADIR="data"
+export DATADIR="data/2018-07-10"
```

Comment:
I think it's best to leave this as `export DATADIR="data/2018-07-10"`. I set the appropriate value in `.local.envrc`, which doesn't get committed, and overrides the value here.